docs/sphinx/omero-pyramid/index.rst

```diff
@@ -0,0 +1,22 @@
+Internal OMERO pyramid format v1.0.0
```

Comment:
For consistency with the rest of the manual, could this be entitled "The OMERO Pyramid format" and move the version to the main text?